stable/cachet/README.md

```diff
@@ -88,6 +88,7 @@ helm install my-release deliveryhero/cachet -f values.yaml
 | ingress.annotations | object | `{}` |  |
 | ingress.enabled | bool | `false` |  |
 | ingress.host | string | `"chart-example.local"` |  |
+| ingress.ingressClassName | string | `"ingressClass"` | Set ingressClass for Ingress when needed |
```

Comment:
This line is removed by helm docs because the value in values.yaml is commented. This is why the CI is failing. I'll suggest a fix in values.yaml.